sample/build.gradle

```diff
@@ -32,6 +32,11 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
+    lint {
+        // This lint check is crashing lint on Compose 1.3.0-alpha01
+        disable "DialogFragmentCallbacksDetector"
```

Comment:
This looks like https://issuetracker.google.com/issues/237567009, should be fixed with `fragment` `1.5.1`?

Reply:
Awesome, thank you!